### PR TITLE
Fair end date

### PR DESCRIPTION
**Pull request by chrisroadmap**
Flexibility to use a non-2100 end date for FaIR, which is taken from the last year of the suppled emissions dataframe.

- [x] Tests added
- [x] Documentation added
- [x] Example added (in the documentation, to an existing notebook, or in a new notebook)
- [x] Description in ``CHANGELOG.rst`` added (single line such as: ``(`#XX <https://github.com/openscm/openscm-runner/pull/XX>`_) Added feature which does something``)

### CODE REVIEW

**Comment by znicholls** [comment]
can you rebase onto https://github.com/openscm/openscm-runner/pull/17 please so we use a more up to date backend (which is way faster)?

**Comment by chrisroadmap** [comment]
> can you rebase onto #17 please so we use a more up to date backend (which is way faster)?

Was going to wait until merged into master - is this not imminent?

**Comment by znicholls** [comment]
> is this not imminent?

Kind of up to you, it depends on https://github.com/OMS-NetZero/FAIR/pull/83

**Comment by chrisroadmap** [comment]
> > is this not imminent?
> 
> Kind of up to you, #17 depends on [OMS-NetZero/FAIR#83](https://github.com/OMS-NetZero/FAIR/pull/83)

Right that pull 83 is merged, I released as 1.6.0c3, do you want to merge #17 into master or should I rebase this?

**Comment by znicholls** [comment]
> Right that pull 83 is merged, I released as 1.6.0c3, do you want to merge #17 into master or should I rebase this?

Great. #17 is merged so please rebase onto master.

**Comment by chrisroadmap** [comment]
Ready to go

File: src/openscm_runner/adapters/fair_adapter/fair_adapter.py
Comment (znicholls) [inline src/openscm_runner/adapters/fair_adapter/fair_adapter.py:54]: Do you want to fix this before merging?

**Review by znicholls — APPROVED** [review]
lgtm, only one question

File: src/openscm_runner/adapters/fair_adapter/fair_adapter.py
Comment (chrisroadmap) [reply src/openscm_runner/adapters/fair_adapter/fair_adapter.py]: Actually let's just delete as an error will be thrown further down the code and I don't want to complicate the possibility of running sub-yearly.

**Comment by znicholls** [comment]
Sounds good

**Pull request merged**
